GliaWidgets/Public/Glia/Glia+EngagementLauncher.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+extension Glia {
+    /// Retrieves an instance of `EngagementLauncher`
+    ///
+    /// - Parameters:
+    ///   - queueIds: A list of queue IDs to be used for the engagement launcher. When nil, the default queues will be used.
+    ///
+    /// - Returns:
+    ///   - `EngagementLauncher` instance
+    ///
```

Comment:
Looks like these extra slashes are not common in the code base. This is probably a good candidate for linter rule.

```suggestion
```